invenio_i18n/templates/invenio_i18n/macros/language_selector.html

```diff
@@ -0,0 +1,23 @@
+
+{% macro language_render() %}
+
+  {% for language_item in config['I18N_LANGUAGES'] %}
+    {{ caller("javascript:document.getElementById('form_invenio_i18n_lang_{0}').submit();".format(language_item[0]), language_item[1]) }}
+  {% endfor %}
+
+  {% for language_item in config['I18N_LANGUAGES'] %}
+    <form id="form_invenio_i18n_lang_{{ language_item[0] }}"
```

Comment:
@jirikuncar I have applied that way but since it is using the ``lang_code`` from query string, I have adapted the ``set_lang`` method in ``views.py`` allowing both of query string and URL parameter.